rfcs/py36.md

```diff
@@ -0,0 +1,28 @@
+# End support for Python 3.6
+
+## Summary
+
+[Support for Python 3.6](https://devguide.python.org/#status-of-python-branches)
+ends on 2021-12-23. When official support has ended, web-platform-tests will stop
+being testing with Python 3.6, and depending on Python 3.7 features will be allowed.
+
+## Details
+
+The immediate motivation for this is that Azure Pipelines no longer supports
+Python 3.6 on macOS 11, meaning that we can no longer easily test Python 3.6 on macOS.
+In order to test the same minimum Python version on all platforms, the version used on
+Taskcluster also needs to be bumped to 3.7, which amounts to ending support for 3.6.
+
+Timeline:
+
+- Immediately: Azure Pipelines on macOS and Windows is upgraded from Python 3.6 to 3.7
+- Between Dec 6 and Dec 17: Taskcluster is upgraded from Python 3.6 to 3.7
+- After Dec 23: Support for testing Python 3.6 is removed throughout WPT
+
+## Risks
+
+Some downstream users of WPT may still be using Python 3.6. TODO, confirm with:
+
+- [ ] Chromium
+- [ ] Gecko
+- [ ] WebKit
```

Comment:
I can't speak for the non-Apple ports, but I believe the situation for the Apple one should be that 3.7 at a minimum is used (as that is what is included with Xcode 11, as used on Catalina).

Reply:
That's great to know, thanks! Do you know which non-Apple ports also run WPT and should be checked? I know about WebKitGTK, but others?

Reply:
For WebKitGTK/WPE, I understand we promise to support Ubuntu Focal Fossa, which is on 3.8, and Debian Buster, which is on 3.7. I'm vaguely aware of Sony maintaining a port as well, but I don't know if they run WPT.